engine/src/flutter/impeller/display_list/aiks_dl_blend_unittests.cc

```diff
@@ -923,5 +923,26 @@ TEST_P(AiksTest, AdvancedBlendColorFilterWithDestinationOpacity) {
   ASSERT_TRUE(OpenPlaygroundHere(builder.Build()));
 }
 
+TEST_P(AiksTest, EmulatedAdvancedBlendRestore) {
+  DisplayListBuilder builder;
+
+  builder.DrawPaint(DlPaint(DlColor::kWhite()));
+  builder.Save();
+  builder.ClipRect(DlRect::MakeLTRB(101, 101, 399, 301));
+  builder.ClipRect(DlRect::MakeLTRB(100, 100, 400, 300));
```

Comment:
Isn't this a NOP?

Reply:
The right edge is slightly bigger ... but this is just confusing. Let me remove this, as a single clip was sufficicient to demonstrate the bug.